addons/html_editor/static/src/core/delete_plugin.js

```diff
@@ -88,21 +91,28 @@ export class DeletePlugin extends Plugin {
     deleteSelection(selection = this.shared.getEditableSelection()) {
         // @todo @phoenix: handle non-collapsed selection around a ZWS
         // see collapseIfZWS
+
+        // Normalize selection
+        selection = this.shared.setSelection(selection);
```

Comment:
do you need a deep normalize ? (now getEditableSelection return a soft normalize selection)